controllers/operands/csv_test.go

```diff
@@ -0,0 +1,91 @@
+package operands
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	csvv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	hcov1beta1 "github.com/kubevirt/hyperconverged-cluster-operator/api/v1beta1"
+	"github.com/kubevirt/hyperconverged-cluster-operator/controllers/common"
+	"github.com/kubevirt/hyperconverged-cluster-operator/controllers/commontestutils"
+	"github.com/kubevirt/hyperconverged-cluster-operator/pkg/components"
+	hcoutil "github.com/kubevirt/hyperconverged-cluster-operator/pkg/util"
+)
+
+var _ = Describe("CSV Operand", func() {
+	var (
+		hco *hcov1beta1.HyperConverged
+		req *common.HcoRequest
+		ci  *commontestutils.ClusterInfoMock
+	)
+
+	BeforeEach(func() {
+		hco = commontestutils.NewHco()
+		req = commontestutils.NewReq(hco)
+		ci = &commontestutils.ClusterInfoMock{}
+	})
+
+	Context("UninstallStrategy is missing", func() {
+		It("should set console.openshift.io/disable-operand-delete to true", func() {
+			foundResource := ensure(req, hco, ci)
+			Expect(foundResource.Annotations[components.DisableOperandDeletionAnnotation]).To(Equal("true"))
```

Comment:
Nit: In order to get better error log from gomega (in case of failure), consider using this instead:
```suggestion
			Expect(foundResource.Annotations).To(HaveKeyWithValue(components.DisableOperandDeletionAnnotation, "true"))
```